learning_resources/fixtures/offered_by.json

```diff
@@ -25,6 +42,11 @@
       "name": "Bootcamps",
       "code": "bootcamps",
       "professional": true,
+      "offerings": ["Courses", "Programs"],
+      "audience": ["Entrepreneurs", "Intrapreneurs", "Innovators"],
+      "formats": ["Online", "In Person"],
+      "content_types": ["Professional"],
+      "more_information": ["https://bootcamps.mit.edu/"],
```

Comment:
The brand working group has recommended that we omit Bootcamps from the MVP. 

I think it's probably easiest to just remove the data at this point, but I'm open to other solutions.

Reply:
Any values labeled as "TBD" in the google doc have been added as "(tbd)" here , as discussed in slack.